### DISCUSSION TITLE
shift-left-how-far-is-too-far/

### DISCUSSION THREAD

**Discussion by giscus** [colindembovsky/colindembovsky.github.io#23] [Giscussions]
# shift-left-how-far-is-too-far/

We’ve all heard the mantra to “shift left” - mainly for testing but also for security. Security scanning earlier (lefter 😸) in the process makes sense, but can you shift left too far?

https://colinsalmcorner.com/shift-left-how-far-is-too-far/

**Comment by ClayNelson** [comment]
Love this take. Keeping developer velocity and organizational ability to drive standard activities within a wide range of development workflows in balance is key.
